tasks/scaffold.js

```diff
@@ -75,7 +81,18 @@ module.exports = function(grunt) {
           'symlink:sites',
           'mkdir:files',
           'copy:static'
-        ]);
+        ];
+
+        // We symlink the config directory if one exists.
+        if (createConfig) {
+          tasks.push('symlink:config');
+          grunt.config(['symlink', 'config'], {
+            src: '<%= config.srcPaths.drupal %>/config',
+            dest: drupal.configPath()
+          });
```

Comment:
This code works just fine, but it does read as out of order to add the task before defining it.

Reply:
✅